public/main.js

```diff
@@ -1,16 +1,33 @@
-function fetchGET(url, cb) {
-    var xhr = new XMLHttpRequest();
-    xhr.onreadystatechange = function() {
-        if (xhr.readySate === 4 && xhr.status === 200) {
-            cb(null, JSON.parse(xhr.responseText));
-        } else if (xhr.readyState === 4 && xhr.status !== 200) {
-            cb(xhr.status);
-        } else {
-            cb(null, xhr.responseText);
-        }
-    };
-    xhr.open('GET', url, true);
-    xhr.send();
+function fetchPOST(url, data, callback) {
+  var xhr = new XMLHttpRequest();
+  xhr.onreadystatechange = function() {
+    if (xhr.readyState == 4 && xhr.status !== 200) {
+      callback(xhr.responseText);
+    } else if (xhr.readyState == 4 && xhr.status === 200) {
+      callback(null, xhr.responseText);
+    }
+  };
+  xhr.open('POST', url);
+  xhr.setRequestHeader('Content-Type', 'application/json');
+  xhr.send(JSON.stringify(data));
 }
 
+var buttons = document.querySelectorAll('.article__btn');
```

Comment:
Why are you using `var`s when everywhere else in the project there are `const`s?